torch_xla/csrc/tensor.cpp

```diff
@@ -1573,7 +1575,8 @@ XLATensor::CompilationResult XLATensor::Compile(
                                    std::move(po_data->emission_map));
   for (auto index : coll.indices) {
     ir::Value ir_value = tensors[index].CurrentIrValue();
-    xla::XlaOp root = lowering_ctx.GetOutputOp(ir_value);
+    xla::XlaOp root = lowering_ctx.GetOutputOp(
+        torch::lazy::Output(ir_value.node.get(), ir_value.index));
```

Comment:
hmm.. seems like we used to can implicitly convert `Value` to `Output` but that is no longer true?

Reply:
Yep, due to the `Output` now being of type `torch::lazy` currently after this migration.